### DISCUSSION TITLE
Práctica CSS

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#294] [Q&A]
Buenos días, apenas empezaré la práctica de  CSS. 

¿Alguna página, videos, atributos y demás que me recomienden?

**Comment by doguedogue** [comment]
Muy bien, con todo 🚀

https://flukeout.github.io/
platzi
udemy

**Reply by GloriaCoralCerecedo** [reply]
Gracias

**Comment by hackit92** [comment]
Faztweb.com tiene cosas útiles de css como bootrsap, gridcss y flexbox, igual podrías practicar este último en https://flexboxfroggy.com/#es

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias!

**Comment by JuanSalvador73** [comment]
https://www.youtube.com/watch?v=wZniZEbPAzk  -- Aprende CSS ahora! curso completo GRATIS desde cero - 2hrs

**Comment by carlos-gmx3** [comment]
Uno de los cursos más completos, solo te recomiendo generar tus propios archivos de practicas
Yo apenas he visto 2 horas
https://www.youtube.com/watch?v=OWKXEJN67FE
